partners/index.md

```diff
@@ -0,0 +1,223 @@
+---
+altLangPage: "https://conception.canada.ca/a-propos/apercu-projet.html"
+css:
+   - https://design.canada.ca/css/split-h1.css
+   - https://design.canada.ca/css/custom.css
+   - https://use.fontawesome.com/releases/v5.15.4/css/all.css
+   - css/card.css
+date: 2020-11-20
+dateModified: 2024-01-25
+description: "The Digital Transformation Office works with federal institutions to make Canada.ca content easier to find and understand."
+title: Working with partners to improve task success on Canada.ca
+---
+<p>The Digital Transformation Office works with federal institutions to make Canada.ca content easier to find and understand.</p>
+<p>Our method includes working collaboratively to:</p>
+<ul>
+  <li>set concrete usability targets</li>
+  <li>perform usability tests on specific Government of Canada Services</li>
+  <li>create prototypes for testing</li>
+  <li>refine prototypes until they achieve at least a 20% improvement in task success</li>
+</ul>
+<h2>Past projects with our partners</h2>
+<div class="row wb-eqht mrgn-tp-md wb-eqht mrgn-tp-lg">
+  <div class="col-xs-12 col-sm-6 col-md-4 mrgn-bttm-lg">
+    <div class="card hght-inhrt"><a href="{{ site.url }}/partners/project-01.html" aria-label="Orientation and wayfinding"><img src="images/blank-blue-lm.png" class="img-responsive" alt="">
+      <div class="card-container">
+        <h3>Orientation and wayfinding</h3>
+        <p class="mrgn-tp-lg"><strong>Purpose</strong>: Understand how people navigate Canada.ca. The findings have led to several design changes to improve navigation throughout Government of Canada websites.</p>
+        <p><strong>Date</strong>: 2021</p>
+        <p><strong>Partners</strong>: Agriculture and Agri-Food Canada; Immigration, Refugees and Citizenship Canada; Veterans Affairs Canada and Statistics Canada</p>
+      </div>
+      </a></div>
+  </div>
+  <div class="col-xs-12 col-sm-6 col-md-4 mrgn-bttm-lg">
+    <div class="card hght-inhrt"><a href="{{ site.url }}/partners/project-02.html" aria-label="Medical devices"><img src="images/blank-blue-lm.png" class="img-responsive" alt="">
+      <div class="card-container">
+        <h3>Medical devices</h3>
+        <p class="mrgn-tp-lg"><strong>Purpose</strong>: Make it easier for stakeholders subject to Canada’s medical device regulations find and understand regulatory requirements.</p>
```

Comment:
Missing a 'to' between 'regulations' and 'find' ... change to: Make it easier for stakeholders subject to Canada's medical device regulations to find and understand regulatory requirements.

Reply:
Fixed.